Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -0,0 +1,1291 @@
+
+        [SerializeField]
+        private float scaleMinimum = 0.2f;
+
+        [Header("Wireframe")]
+        [SerializeField]
+        private bool wireframeOnly = false;
+
+        public bool WireframeOnly
+        {
+            get { return wireframeOnly; }
+            set
+            {
+                if (wireframeOnly != value)
+                {
+                    wireframeOnly = value;
+                    ResetHandleVisibility();
+                }
+            }
+        }
+
+        [SerializeField]
+        private Vector3 wireframePadding = Vector3.zero;
+
+        [SerializeField]
+        private FlattenModeType flattenAxis = FlattenModeType.DoNotFlatten;
+
+        [SerializeField]
+        private WireframeType wireframeShape = WireframeType.Cubic;
+
+        [SerializeField]
+        private Material wireframeMaterial;
+
+        [Header("Handles")]
+        [Tooltip("Default materials will be created for Handles and Wireframe if none is specified.")]
+        [SerializeField]
+        private Material handleMaterial;
+
+        [SerializeField]
+        private Material handleGrabbedMaterial;
+
+        [SerializeField]
+        private bool showScaleHandles = true;
+
+        public bool ShowScaleHandles
+        {
+            get
+            {
+                return showScaleHandles;
+            }
+            set
+            {
+                if (showScaleHandles != value)
+                {
+                    showScaleHandles = value;
+                    ResetHandleVisibility();
+                }
+            }
+        }
+
+        [SerializeField]
+        private bool showRotateHandles = true;
+
+        public bool ShowRotateHandles
+        {
+            get
+            {
+                return showRotateHandles;
+            }
+            set
+            {
+                if (showRotateHandles != value)
+                {
+                    showRotateHandles = value;
+                    ResetHandleVisibility();
+                }
+            }
+        }
+
+        [SerializeField]
+        private float linkRadius = 0.005f;
+
+        [SerializeField]
+        private float ballRadius = 0.035f;
+
+        [SerializeField]
+        private float cornerRadius = 0.03f;
+
+        #endregion Serialized Fields
+
+        private bool isActive = false;
+
+        public bool IsActive
+        {
+            get
+            {
+                return isActive;
+            }
+            set
+            {
+                if (isActive != value)
+                {
+                    if (value)
+                    {
+                        CreateRig();
+                        rigRoot.SetActive(true);
+                    }
+                    else
+                    {
+                        DestroyRig();
+                    }
+
+                    isActive = value;
+                }
+            }
+        }
+
+        #region Constants
+
+        private const int LTB = 0;
```

Comment:
are there more descriptive names for these constants?

Reply:
👍  changed to English readable: LeftTopBack etc